src/components/weatherObject.js

```diff
@@ -0,0 +1,32 @@
+function WeatherInfo(data){
+	this.location = `${data.location.name} , ${data.location.country}`;
+	this.temperature = data.current.temperature;
+	const newTime = data.location.localtime.split(" ")[1];
+	this.time = newTime;
+	this.feelsLike = data.current.feelslike;
+	this.weatherDescriptions = data.current.weather_descriptions;
+	this.windDir = data.current.wind_dir;
+	this.windSpeed = data.current.wind_speed;
+	this.pressure = data.current.pressure;
+}
```

Comment:
```suggestion
const getWeatherInfo = (data) => {
  const time = data.location.localtime.split(" ")[1];

  return {
	location: `${data.location.name} , ${data.location.country}`,
	temperature: data.current.temperature,
	time,
	feelsLike: data.current.feelslike,
	weatherDescriptions: data.current.weather_descriptions,
	windDir: data.current.wind_dir,
	windSpeed: data.current.wind_speed,
	pressure: data.current.pressure,
  };
}
```
Нет необходимости для использования функции конструктора